spec/resources/example-condition.ttl

```diff
@@ -0,0 +1,66 @@
+@prefix dbo: <http://dbpedia.org/ontology/> .
+@prefix grel: <http://users.ugent.be/~bjdmeest/function/grel.ttl#> .
```

Comment:
`grel` is still this URI or should we agree on the new one?

Reply:
using the example.com/fns for now